homeassistant/components/openweathermap/__init__.py

```diff
@@ -103,13 +106,13 @@ def _filter_domain_configs(elements, domain):
     return list(filter(lambda elem: elem["platform"] == domain, elements))
```

Comment:
Please type `_filter_domain_configs` too.

Reply:
The function seems not to be used. I am struggling on finding out the types.

Reply:
Good, then we can remove it. :+1: